torchci/pages/benchmark/[suite]/[compiler]/[[...page]].tsx

```diff
@@ -428,6 +428,11 @@ function ModelPanel({
                     return "";
                   }
 
+                  // It didn't error in the past, but now it does error
```

Comment:
I think the preview still doesn't high light it correctly like what you describe https://torchci-git-fork-davidberard98-red-speedup-0-fbopensource.vercel.app/benchmark/torchbench/inductor_dynamic?stopTime=Mon%2C%2003%20Jul%202023%2022%3A28%3A15%20GMT&startTime=Mon%2C%2026%20Jun%202023%2022%3A28%3A15%20GMT&mode=training&dtype=amp&lCommit=7bc181d374a54db4b1556d263be965f6a848eecb&rCommit=d4e51511a0e471072359800b03e1b63f2ef2b3f6&granularity=hour&rBranch=main&lBranch=main

You could tweak the return value on like 428 a bit, for example:

```
// l means the new value and r means the old value, there is no change and no need to highlight anything even
// if the new value (l) is still 0
if (l === r) {
  return ""
}

// The new value (l) is 0, and the old value (r) is non-zero
if (l === 0) {
  return styles.error;
}
```

The code uses the convention that `l` means the new value and `r` means the old value.  So, you would need to check for `l` instead.  For historical context, the dashboard used to have `new value (l) <- old value (r)`, but now it uses `old value (r) -> new value (l)`.  I should have refactor everything so that `l` and `r` swaps :)

Reply:
ah thanks for the context on l vs r